auth/test/TestMT50.m

```diff
@@ -0,0 +1,29 @@
+classdef TestMT50 < matlab.unittest.TestCase
+
+    methods (Test)
+
+        function test_encode_decode(testcase)
+            HMAC1 = [uint8(3); uint8(5)];
+            HMAC2 = [uint8(7); uint8(11)];
+            HMAC3 = [uint8(13); uint8(17)];
+            HMAC4 = [uint8(19); uint8(23)];
+            HMAC5 = [uint8(29); uint8(31)];
+            hp = [uint8(11); uint8(45); uint8(104); uint8(155); uint8(1); ...
```

Comment:
nitpick: shorter line: uint8([11; 45; 104; ...).

If you editing this file in sublime...it will be quick. See the following links:

https://www.sublimetext.com/docs/3/multiple_selection_with_the_keyboard.html
https://www.youtube.com/watch?v=czQipWJA8EU